1_Models/test_model.js

```diff
@@ -60,10 +59,37 @@ const fetchCommentsByArticleId = (article_id) => {
     });
 };
 
+const addComments = (article_id, username, body) => {
+  return db
+    .query(
+      `
+    SELECT * FROM articles WHERE article_id = $1;
```

Comment:
this is not necessary because article_id is a foreign key on the comments table and if the foreign key does not exist psql will throw an error.